K2Bridge/Models/Response/DateHistogramBucket.cs

```diff
@@ -30,5 +33,11 @@ public class DateHistogramBucket : IBucket
         /// </summary>
         [JsonProperty("key_as_string")]
         public string KeyAsString { get; set; }
+
+        /// <summary>
+        /// Gets or Sets fdsfd.
+        /// </summary>
+        [JsonIgnore]
+        public Dictionary<string, List<long>> Aggs { get; set; }
```

Comment:
This should be a floating-point type. ES does return floats for aggregations like average.